src/Features/Core/Portable/Completion/Providers/AbstractAwaitCompletionProvider.cs

```diff
@@ -30,11 +30,9 @@ internal abstract class AbstractAwaitCompletionProvider : LSPCompletionProvider
 
         public sealed override async Task ProvideCompletionsAsync(CompletionContext context)
         {
-            var document = context.Document;
-            var position = context.Position;
-            var cancellationToken = context.CancellationToken;
+            var (document, position, cancellationToken) = (context.Document, context.Position, context.CancellationToken);
```

Comment:
not a fan :)   

if you wanted this you coudl make CompletionContext deconstructable.